CMakeLists.txt

```diff
@@ -80,26 +80,23 @@ set_target_properties (OpenCL PROPERTIES VERSION "1.2" SOVERSION "1")
 if (WIN32)
     target_link_libraries (OpenCL cfgmgr32.lib)
 
-    option (OPENCL_ICD_LOADER_REQUIRE_WDK "Build with D3DKMT support, which requires the Windows WDK." ON)
-    if (OPENCL_ICD_LOADER_REQUIRE_WDK)
-        if(DEFINED ENV{WDKContentRoot})
-            file(GLOB D3DKMT_HEADER "$ENV{WDKContentRoot}/Include/*/km/d3dkmthk.h")
-        else()
-            file(GLOB D3DKMT_HEADER "$ENV{HOMEDRIVE}/Program Files */Windows Kits/10/Include/*/km/d3dkmthk.h")
-        endif()
+    if(DEFINED ENV{WDKContentRoot})
```

Comment:
It looks like the WDK is still required after these changes, specifically for the WDK headers, which is unfortunate.

Can the OpenCL ICD loader use its own header instead, similar to the Vulkan loader?

* https://github.com/KhronosGroup/Vulkan-Loader/commit/7557a5753409f89d78eaaf23367cecc60574b3be

I think this would break all dependencies on the WDK?